tests/fixtures/single_run/config/sos_model_runs/20170918_energy_water.yml

```diff
@@ -0,0 +1,15 @@
+name: 'energy_water'
```

Comment:
This should be the same as the filename in the current DatafileInterface implementation.